networkx/algorithms/centrality/tests/test_group.py

```diff
@@ -6,6 +6,9 @@
 import pytest
 import networkx as nx
 
+np = pytest.importorskip("numpy")
+pd = pytest.importorskip("pandas")
+
```

Comment:
I think these lines need to be moved within the `TestProminentGroup` class, otherwise all of the tests in the file will be skipped if numpy or pandas aren't present.